airbyte-integrations/connectors/source-klaviyo/acceptance-test-config.yml

```diff
@@ -5,6 +5,8 @@ acceptance_tests:
       empty_streams:
       - name: flows
         bypass_reason: "the 'trigger' object may mutate often"
+      - name: profiles
+        bypass_reason: "the airbyte team will need to add these to integration tests"
```

Comment:
@Wittiest for this change please remove this line. Integration tests must validate the new stream is working.